web/ralucarbc/javascript/squares-sequence/squares-sequence.js

```diff
@@ -0,0 +1,30 @@
+const h2Instructions = document.querySelector("#instructions");
+const divContent = document.querySelector("#content");
+const divOutput = document.querySelector("#output");
+const divFunction = document.querySelector("#function");
+
+h2Instructions.textContent = `Complete the function that returns an array of length n, starting with the given number x and the squares of the previous number. If n is negative or zero, return an empty array/list.`;
```

Comment:
I've seen this throughout your commits. The use of ticks (Template String) and concatenation as well. 
If ES6 is permitted, feel free to use just Template String (Ex: `This is my string bit to which I add my ${variable}`)
Else, use old school string concat (Ex: "This is my string to which I add my " + variable)